### ISSUE TITLE
Text Color option is missing in Quote Block.

### ISSUE THREAD

**Issue by annbingle** [Automattic/wp-calypso#57412]
### Quick summary

The user is trying to change the color of their text inside the Quote block. However, they don't see the option to change the color on the top tool bar nor the actual Block settings on the right panel.

I recreated it on my Test Sites; one on simple site and the other one is in my AT test site

I don't see the Text color for Quotes block on my simple site too.

<img width="1263" alt="Screen Shot on 2021-10-28 at 10-55-11" src="https://user-images.githubusercontent.com/40605613/139178266-2fa8030a-56ec-42d2-b28b-8cff1709fb8b.png">

However, it is visible on my AT site.
<img width="1263" alt="Screen Shot on 2021-10-28 at 10-55-11" src="https://user-images.githubusercontent.com/40605613/139178681-8b804fa6-d7c1-4eaa-b831-ea09d683ca82.png">

### Steps to reproduce

1. Insert the "Quote Block" to your post  or page.
2. Enter texts inside the block itself. Click on the down arrow on the top toolbar of the Quotes block and the "Highlight" option is missing.
3. You can also go the Block settings on the right panel. The Color option isn't in the block settings too.

### What you expected to happen

There should be the Highlights option to change the Text color or add Background color to the texts inside the block. Additionally, there should be Color option on the Block settings on the right Panel Block Settings.

### What actually happened

**_Highlights_** option is missing on the top toolbar and the **_Colors_** option on the Block Settings on the right panel is missing.

### Context

I have provided CSS Code to change the text color as a Workaround.

### Simple, Atomic or both?

Simple

### Theme-specific issue?

No, the user is using Dara and I am using Dynamic News theme on my test page.

### Available workarounds?

Yes, easy to implement

### Workaround details

I provided a CSS code as a temporary workaround. I specified the post id in the code so it will not affect other Quotes block from their other posts.

```
/* Changes color of the text inside the Quotes Block | 21734990-hc (AB) */
.page-id-3128 .wp-block-quote {
color: #cd0dda;
}
```

**Comment by dpasque** [comment]
Hi @annbingle! 👋 

I have been having a hard time reproducing the dropdown piece, and was wondering if you had any more information?

On both Simple/Atomic, I am seeing the `Highlight` option in the drop down menu within a quote block, and that allows changing the text color.

![Screen Shot 2021-10-28 at 3 36 31 PM](https://user-images.githubusercontent.com/26772342/139332987-733f2621-ebd9-4714-8783-f7b176b24111.png)

However, I am seeing the sidebar discrepancy you mentioned: color as an _sidebar_ option is only available on Atomic.

**Comment by ivan-ottinger** [comment]
We did look at the issue with @arcangelini and @StefanNieuwenhuis today and here are our findings:

- the `Highlight` dropdown is displaying correctly on both Simple and Atomic sites - as @dpasque mentioned in the previous comment
- the `Colors` section in the sidebar is displayed on Atomic sites only because it is coming from CoBlocks plugin that is preinstalled and activated on Atomic sites. If we deactivate the CoBlocks plugin, the `Color` section in the sidebar gets removed.

In other words, things seem to be working as expected. :) I will close the issue for now, but feel free to reopen in case we missed something. Thank you.
